src/config.c

```diff
@@ -208,6 +208,29 @@ int parse_mac_address (char *target, const char *arg)
 	return 0;
 }
 
+int parse_serial_num(char *target, const char *arg)
+{
+	char ch;
+	if(strlen(arg) == 0)
+	{
+	    ParodusError("Empty serial number\n");
+	    return -1;
```

Comment:
Lets not return error when it is empty, just log error and set to default "unknown"